### PR TITLE
Feature/replat 3983 add alt tag to images

### PR DESCRIPTION
**Pull request by mikefieldmay**
Jira ticket: https://nidigitalsolutions.jira.com/browse/REPLAT-3983

Add alt text to images for article pages

### CODE REVIEW

**Comment by parkjh** [comment]
@mikefieldmay is this WIP?

**Comment by mikefieldmay** [comment]
@parkjh yeah, sorry should have marked it as such

File: packages/image/src/modal-image/modal-image-prop-types.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const modalPropTypes = {
 
 export const modalDefaultProps = {
   ...defaultProps,
+  accessibilityLabel: "",
```
Comment (parkjh) [inline packages/image/src/modal-image/modal-image-prop-types.js]: is default value really necessary?

File: packages/article-image/src/inline-image.web.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ const InlineImage = ({ imageOptions, captionOptions }) => {
     <figure style={{ margin: 0 }}>
       <InsetImageStyle key="img">
         <Image
+          alt={caption}
```
Comment (parkjh) [inline packages/article-image/src/inline-image.web.js]: don't you mean `caption.text`?

File: packages/image/src/image-prop-types.js [hunk 3]
```diff
@@ -24,3 +25,8 @@ export const defaultProps = {
   rounded: false,
   uri: null
 };
+
+export const webDefaultProps = {
+  ...defaultProps,
+  accessibilityLabel: ""
```
Comment (parkjh) [inline packages/image/src/image-prop-types.js]: I think as a team we decided against setting component default props... as this was going to be phased out by facebook

Comment (parkjh) [reply packages/image/src/image-prop-types.js]: also why have empty string here?

File: packages/article-image/src/inline-image.web.js (hunk 2 above)
Comment (nehasri89) [reply packages/article-image/src/inline-image.web.js]: no, we get the markup as `caption`

File: packages/image/src/modal-image/modal-image-prop-types.js (hunk 1 above)
Comment (mikefieldmay) [reply packages/image/src/modal-image/modal-image-prop-types.js]: Yeah you're right I don't think it is

File: packages/image/src/image-prop-types.js (hunk 3 above)
Comment (mikefieldmay) [reply packages/image/src/image-prop-types.js]: I'll remove these default props too

**Pull request merged**